content/docs/redhat/acm/submariner/rosa/rosa-submariner.md

```diff
@@ -0,0 +1,706 @@
+
+```sh
+$ kubectl get ns | grep broker
+default-broker                                     Active   6h39m
+rosa-clusters-broker                               Active   13m
+```
+
+* Create the Broker configuration on the hub cluster in the managed-cluster-set-name-broker namespace
+
+```sh
+cat << EOF | kubectl apply -f -
+apiVersion: submariner.io/v1alpha1
+kind: Broker
+metadata:
+     name: submariner-broker
+     namespace: rosa-clusters-broker
+spec:
+     globalnetEnabled: false
+EOF
+```
+
+NOTE: Set the the value of globalnetEnabled to true if you want to enable Submariner Globalnet in the ManagedClusterSet.
+
+* Check the Submariner Broker in the rosa-clusters-broker namespace:
+
+```sh
+kubectl get broker -n rosa-clusters-broker
+NAME                AGE
+submariner-broker   21s
+```
+
+* We don’t need to label the ManagedCluster because it was imported the proper labels within the proper ManagedClusterSet:
+
+* Deploy SubmarinerConfig for the first rosa cluster imported:
+
+```sh
+cat << EOF | kubectl apply -f -
+apiVersion: submarineraddon.open-cluster-management.io/v1alpha1
+kind: SubmarinerConfig
+metadata:
+  name: submariner
+  namespace: $ROSA_CLUSTER_NAME_1
+spec:
+  IPSecNATTPort: 4500
+  NATTEnable: true
+  cableDriver: libreswan
+  loadBalancerEnable: true
+EOF
+```
+
+* Deploy SubmarinerConfig for the second rosa cluster imported:
+
+```sh
+cat << EOF | kubectl apply -f -
+apiVersion: submarineraddon.open-cluster-management.io/v1alpha1
+kind: SubmarinerConfig
+metadata:
+  name: submariner
+  namespace: $ROSA_CLUSTER_NAME_2
+spec:
+  IPSecNATTPort: 4500
+  NATTEnable: true
+  cableDriver: libreswan
+  loadBalancerEnable: true
+EOF
+```
+
+* Deploy Submariner on the first ROSA cluster cluster:
+
+```sh
+cat << EOF | kubectl apply -f -
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: ManagedClusterAddOn
+metadata:
+     name: submariner
+     namespace: $ROSA_CLUSTER_NAME_1
+spec:
+     installNamespace: submariner-operator
+EOF
+```
+
+* Deploy Submariner on the second ROSA cluster cluster:
+
+```sh
+cat << EOF | kubectl apply -f -
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: ManagedClusterAddOn
+metadata:
+     name: submariner
+     namespace: $ROSA_CLUSTER_NAME_2
+spec:
+     installNamespace: submariner-operator
+EOF
+```
+
+* Check the submariner managedclusteraddons status in order to check if submariner is deployed correctly
+
+```sh
+kubectl get managedclusteraddon -A | grep submariner
+rosa-sbmr1      submariner                    True
+rosa-sbmr2      submariner                    True
+```
+
+The Submariner Add-on installation will start, and will take up to 10 minutes to finish.
+
+## Check the Status of the Submariner Networking Add-On 
+
+* Few minutes (up to 10 minutes) after we can check that the app Connection Status and the Agent Status are Healthy:
```

Comment:
You're missing article adjective "A" at the start of this sentence. Additionally, a single bullet section needs no bullets. :)

Reply:
done :)